python/cuml/dask/neighbors/kneighbors_regressor.py

```diff
@@ -198,7 +198,7 @@ def predict(self, X, convert_dtype=True):
 
         comms.destroy()
 
-        out = to_output(out_futures, self.datatype)
+        out = to_output(out_futures, self.datatype).squeeze()
         out_i = to_output(out_i_futures, self.datatype)
         out_d = to_output(out_d_futures, self.datatype)
         return out, out_i, out_d
```

Comment:
We should probably not be returning the indices and distances from `predict()`. Scikit-learn just returns the predictions: https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html. Also, since the `KNeighborsRegressor/Classifier` Dask wrappers extend the `NearestNeighbors` wrapper, the `kneighbors()` function is still available for users if they need those.